### PR TITLE
Sync dev with 4.2

### PR DESCRIPTION
**Pull request by SaikrishnaBairamoni**
# PR Details
This PR is to merge release cabin into develop
## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: freightliner_cascadia_2012_dot_10004/docker-compose-background.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ version: '2'
 
 services:
   web-ui:
-    image: usdotfhwastoldev/carma-web-ui:develop
+    image: usdotfhwastol/carma-web-ui:carma-system-4.2.0
```
Comment (kjrush) [inline freightliner_cascadia_2012_dot_10004/docker-compose-background.yml:20]: Should still be develop

File: lexus_rx_450h_2019/docker-compose-background.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ version: '2'
 
 services:
   web-ui:
-    image: usdotfhwastoldev/carma-web-ui:develop
+    image: usdotfhwastol/carma-web-ui:carma-system-4.2.0
```
Comment (kjrush) [inline lexus_rx_450h_2019/docker-compose-background.yml:20]: Should be develop

**Review by kjrush — CHANGES_REQUESTED** [review]

File: freightliner_cascadia_2012_dot_10004/docker-compose-background.yml (hunk 1 above)
Comment (SaikrishnaBairamoni) [reply freightliner_cascadia_2012_dot_10004/docker-compose-background.yml]: Updated to develop

File: lexus_rx_450h_2019/docker-compose-background.yml (hunk 2 above)
Comment (SaikrishnaBairamoni) [reply lexus_rx_450h_2019/docker-compose-background.yml]: updated to develop

**Review by kjrush — APPROVED** [review]

**Pull request merged**